lib/ansible/modules/cloud/amazon/elb_target_group.py

```diff
@@ -119,7 +120,7 @@
     version_added: 2.5
   targets:
     description:
-      - A list of targets to assign to the target group. This parameter defaults to an empty list. Unless you set the 'modify_targets' parameter then
+      - A list of targets to assign to the target group. This parameter defaults to an empty list. Unless you set the 'purge_targets' parameter then
```

Comment:
s/This parameter defaults to an empty list./This parameter defaults to None.

The line below should be edited too since to drain all targets it must be set to an empty list to use in conjunction with purge_targets.

Reply:
It still does not default to an empty list, it defaults to None. It will appear null in the invocation.